.ci/Jenkinsfile

```diff
@@ -343,6 +345,19 @@ pipeline {
             }
           }
         }
+        stage('Publish to npm') {
```

Comment:
The `stage('Opbeans')` above indirectly depends on the package already having been published to npm because the [`bump-version.sh` script it is running does `npm install elastic-apm-node@$THE-PUBLISHED-VERSION`](https://github.com/elastic/opbeans-node/blob/f2cdbe3a68c1ba4ea5f10907bac549320218173f/.ci/bump-version.sh#L14). So I think this should move the 'Opbeans' stage to the bottom of this `stage('Release')` block.

Reply:
@pazone There is still this to do.

Reply:
moved to the end of release stage